pkg/converter/tool/feature_test.go

```diff
@@ -13,11 +13,18 @@ import (
 )
 
 func TestFeature(t *testing.T) {
-	features := Features{FeatureTar2Rafs}
-	require.True(t, features.Contains(FeatureTar2Rafs))
-
-	features.Remove(FeatureTar2Rafs)
-	require.False(t, features.Contains(FeatureTar2Rafs))
+	f1 := Features{FeatureTar2Rafs}
+	require.True(t, f1.Contains(FeatureTar2Rafs))
+
+	f1.Remove(FeatureTar2Rafs)
+	require.False(t, f1.Contains(FeatureTar2Rafs))
+
+	f2 := Features{FeatureTar2Rafs, FeatureBatchSize}
+	f3 := Features{FeatureTar2Rafs, FeatureBatchSize}
+	f4 := Features{FeatureBatchSize, FeatureTar2Rafs}
+	require.True(t, f2.Equals(f3))
+	require.False(t, f2.Equals(f4))
```

Comment:
The features is a set, should be no order required.

Reply:
Changed.